### ISSUE TITLE
Cannot display under Atom/Juno (Julia v0.7.0)

### ISSUE THREAD

**Issue by BoundaryValueProblems** [sglyon/PlotlyJS.jl#214]
I just added the current master branch of PlotlyJS.jl https://github.com/sglyon/PlotlyJS.jl/commit/2a1c12889ed1232e2d09005562d81ce15ae854f9. Under Atom/Juno, I got the following error using genuine PlotlyJS functions (not using Plots.jl). *Note that if I run it on Terminal without Atom/Juno, it just works as expected.*
```
Error showing value of type PlotlyJS.SyncPlot:
ERROR: MethodError: no method matching show(::IOContext{Base.GenericIOBuffer{Array{UInt8,1}}}, ::Type{MIME{Symbol("text/html")}}, ::PlotlyBase.Plot{GenericTrace{Dict{Symbol,Any}}}; js=:local)
Closest candidates are:
  show(::IO, ::MIME{Symbol("text/plain")}, ::PlotlyBase.Plot) at /Users/xxx/.julia/packages/PlotlyBase/prOMV/src/PlotlyBase.jl:42 got unsupported keyword argument "js"
  show(::IO, ::MIME{Symbol("text/plain")}, ::Any) at sysimg.jl:195 got unsupported keyword argument "js"
  show(::IO, ::DataType) at show.jl:437 got unsupported keyword argument "js"
  ...
Stacktrace:
 [1] show(::IOContext{Base.GenericIOBuffer{Array{UInt8,1}}}, ::MIME{Symbol("application/juno+plotpane")}, ::PlotlyBase.Plot{GenericTrace{Dict{Symbol,Any}}}) at /Users/xxx/.julia/packages/PlotlyJS/Z7d5n/src/display.jl:37
 [2] show(::IOContext{Base.GenericIOBuffer{Array{UInt8,1}}}, ::MIME{Symbol("application/juno+plotpane")}, ::PlotlyJS.SyncPlot) at /Users/xxx/.julia/packages/PlotlyJS/Z7d5n/src/display.jl:58
 [3] show(::IOContext{Base.GenericIOBuffer{Array{UInt8,1}}}, ::String, ::PlotlyJS.SyncPlot) at ./multimedia.jl:79
 [4] displayinplotpane(::PlotlyJS.SyncPlot) at /Users/xxx/.julia/packages/Atom/Pab0Z/src/display/showdisplay.jl:26
 [5] display(::Atom.JunoDisplay, ::PlotlyJS.SyncPlot) at /Users/xxx/.julia/packages/Atom/Pab0Z/src/display/showdisplay.jl:55
 [6] display(::PlotlyJS.SyncPlot) at ./multimedia.jl:287
 [7] #invokelatest#1 at ./essentials.jl:691 [inlined]
 [8] invokelatest at ./essentials.jl:690 [inlined]
 [9] print_response(::IO, ::Any, ::Any, ::Bool, ::Bool, ::Any) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:154
 [10] print_response(::REPL.AbstractREPL, ::Any, ::Any, ::Bool, ::Bool) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:139
 [11] (::getfield(REPL, Symbol("#do_respond#40")){Bool,getfield(Atom, Symbol("##162#163")),REPL.LineEditREPL,REPL.LineEdit.Prompt})(::Any, ::Any, ::Any) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:708
 [12] #invokelatest#1 at ./essentials.jl:691 [inlined]
 [13] invokelatest at ./essentials.jl:690 [inlined]
 [14] run_interface(::REPL.Terminals.TextTerminal, ::REPL.LineEdit.ModalInterface, ::REPL.LineEdit.MIState) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/REPL/src/LineEdit.jl:2261
 [15] run_frontend(::REPL.LineEditREPL, ::REPL.REPLBackendRef) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:1029
 [16] run_repl(::REPL.AbstractREPL, ::Any) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/REPL/src/REPL.jl:191
 [17] (::getfield(Base, Symbol("##831#833")){Bool,Bool,Bool,Bool})(::Module) at ./logging.jl:311
 [18] #invokelatest#1 at ./essentials.jl:691 [inlined]
 [19] invokelatest at ./essentials.jl:690 [inlined]
 [20] macro expansion at ./logging.jl:308 [inlined]
 [21] run_main_repl(::Bool, ::Bool, ::Bool, ::Bool, ::Bool) at ./client.jl:340
 [22] exec_options(::Base.JLOptions) at ./client.jl:252
 [23] _start() at ./client.jl:432
```

**Comment by sglyon** [comment]
Thank you -- sorry for the mess right now.

Things will smooth out very soon if you hang with us!

**Comment by sglyon** [comment]
Version 0.12.0 of this package has been released and is compatible with Julia 1.0
